src/plugins/recurring-issues/index.js

```diff
@@ -15,9 +15,40 @@ const moment = require("moment-timezone");
 async function getReleaseIssueBody(releaseDate) {
     return `
 
-The scheduled release on ${releaseDate.format("dddd, MMMM Do, YYYY")} is assigned to:
+The next scheduled release will occur on ${releaseDate.format("dddd, MMMM Do, YYYY")}.
 
-* (needs volunteer)
+## Release Day Checklist
+
+- [ ] Remove the 'tsc agenda' label on this issue
+- [ ] Review open pull requests and merge any that are [ready](https://eslint.org/docs/maintainer-guide/pullrequests#when-to-merge-a-pull-request)
+- [ ] Start the release on [Jenkings](https://jenkins.eslint.org)
+- [ ] Update the release blog post with a "Highlights" section
+- [ ] Make a release announcement on Twitter
+- [ ] Make a release announcement in the Discord '#announcements' channel
+- [ ] Add a comment to this issue saying the release is out
+- [ ] Add the 'patch release pending' label to this issue
+
+## Monday After Release Day Checklist
+
+- [ ] Check the issues list for any regression issues
+
+## No Regressions Checklist
+
+- [ ] Remove the 'patch release pending' label from this issue
+- [ ] Close this issue
+
+## Patch Release Checklist
+
+- [ ] Resolve the regression by merging any necessary fixes
+- [ ] Start the release on [Jenkings](https://jenkins.eslint.org)
+- [ ] Update the release blog post with a "Highlights" section
+- [ ] Make a release announcement on Twitter
+- [ ] Make a release announcement in the Discord '#announcements' channel
+- [ ] Add a comment to this issue saying the release is out
+- [ ] Remove the 'patch release pending' label from this issue
```

Comment:
```suggestion
- [ ] Wait two days and repeat the Two Days After a Release checklist
- [ ] Remove the 'patch release pending' label from this issue
```

Reply:
@btmills does this capture it correctly?

Reply:
```suggestion
- [ ] Wait two days and repeat the Two Days After a Release checklist
```

That'll work! Suggestion changed just to remove the "Remove the ... label" step since that's covered in the "No Regressions" case and we want to keep the label until then.

Reply:
Sounds good